Starting test 1
Traceback (most recent call last):
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 74, in <module>
    main()
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in main
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in <listcomp>
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\tema\producer.py", line 36, in __init__
    super.__init__(super,**kwargs)
TypeError: descriptor '__init__' requires a 'super' object but received a 'type'
TIMEOUT. Test 1 exceeded maximum allowed time of 30
Finished test 1
Test 1:		FAILED
Starting test 2
Traceback (most recent call last):
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 74, in <module>
    main()
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in main
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in <listcomp>
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\tema\producer.py", line 36, in __init__
    super.__init__(super,**kwargs)
TypeError: descriptor '__init__' requires a 'super' object but received a 'type'
TIMEOUT. Test 2 exceeded maximum allowed time of 30
Finished test 2
Test 2:		FAILED
Starting test 3
Traceback (most recent call last):
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 74, in <module>
    main()
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in main
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in <listcomp>
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\tema\producer.py", line 36, in __init__
    super.__init__(super,**kwargs)
TypeError: descriptor '__init__' requires a 'super' object but received a 'type'
TIMEOUT. Test 3 exceeded maximum allowed time of 30
Finished test 3
Test 3:		FAILED
Starting test 4
Traceback (most recent call last):
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 74, in <module>
    main()
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in main
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in <listcomp>
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\tema\producer.py", line 36, in __init__
    super.__init__(super,**kwargs)
TypeError: descriptor '__init__' requires a 'super' object but received a 'type'
TIMEOUT. Test 4 exceeded maximum allowed time of 30
Finished test 4
Test 4:		FAILED
Starting test 5
Traceback (most recent call last):
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 74, in <module>
    main()
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in main
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in <listcomp>
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\tema\producer.py", line 36, in __init__
    super.__init__(super,**kwargs)
TypeError: descriptor '__init__' requires a 'super' object but received a 'type'
TIMEOUT. Test 5 exceeded maximum allowed time of 30
Finished test 5
Test 5:		FAILED
Starting test 6
Traceback (most recent call last):
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 74, in <module>
    main()
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in main
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in <listcomp>
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\tema\producer.py", line 36, in __init__
    super.__init__(super,**kwargs)
TypeError: descriptor '__init__' requires a 'super' object but received a 'type'
TIMEOUT. Test 6 exceeded maximum allowed time of 30
Finished test 6
Test 6:		FAILED
Starting test 7
Traceback (most recent call last):
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 74, in <module>
    main()
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in main
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in <listcomp>
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\tema\producer.py", line 36, in __init__
    super.__init__(super,**kwargs)
TypeError: descriptor '__init__' requires a 'super' object but received a 'type'
TIMEOUT. Test 7 exceeded maximum allowed time of 30
Finished test 7
Test 7:		FAILED
Starting test 8
Traceback (most recent call last):
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 74, in <module>
    main()
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in main
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in <listcomp>
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\tema\producer.py", line 36, in __init__
    super.__init__(super,**kwargs)
TypeError: descriptor '__init__' requires a 'super' object but received a 'type'
TIMEOUT. Test 8 exceeded maximum allowed time of 30
Finished test 8
Test 8:		FAILED
Starting test 9
Traceback (most recent call last):
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 74, in <module>
    main()
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in main
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in <listcomp>
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\tema\producer.py", line 36, in __init__
    super.__init__(super,**kwargs)
TypeError: descriptor '__init__' requires a 'super' object but received a 'type'
TIMEOUT. Test 9 exceeded maximum allowed time of 60
Finished test 9
Test 9:		FAILED
Starting test 10
Traceback (most recent call last):
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 74, in <module>
    main()
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in main
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\test.py", line 56, in <listcomp>
    producers = [Producer(**p_market_config, marketplace=marketplace, daemon=True)
  File "C:\Users\priot\Documents\An 3\Sem II\ASC\asc-public\assignments\1-marketplace\skel\tema\producer.py", line 36, in __init__
    super.__init__(super,**kwargs)
TypeError: descriptor '__init__' requires a 'super' object but received a 'type'
TIMEOUT. Test 10 exceeded maximum allowed time of 60
Finished test 10
Test 10:		FAILED

Run pylint

C:\Users\priot\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\python.exe: No module named pylint
